Lintr Changes

AKktk opened the pull request.

AKktk committed:
Lintr Changes

kkmann commented:
not passing the tests yet...

AKktk committed:
Lintr editing for tests completed

kkmann committed:
applied styler:: to the two markdown vignettes fixing all the whitespace issues

AKktk committed:
Lintr Changes

AKktk committed:
Resolved merge conflict

AKktk committed:
Lintr issues and warnings resolved

AKktk committed:
Resolving merge conflict within util.R

AKktk committed:
Lintr resolutions

kkmann committed:
removed unnecessary # nolints

kkmann committed:
updated docs

kkmann reviewed (approved):
Please avoid # nolint in documentation strings, otherwise it show in the final docs. Otheriwse - great, ready to merge!

kkmann committed:
merge main

Merge branch 'main' into double-check-adherence-to-tidyverse-style-guideline

# Conflicts:
#	R/util.R
